config/locales/views/stories/en.yml

```diff
@@ -21,5 +21,5 @@ en:
         year: Year
         infinity: Infinity
         latest: Latest
-        signin_html: "👋 <a href=\"%{path}\">Sign in</a> for the ability sort posts by <strong>top</strong> and  <strong>latest</strong>."
+        signin_html: "👋 <a href=\"%{path}\">Sign in</a> for the ability to sort posts by <strong>relevant</strong>, <strong>latest</strong>, or <strong>top</strong>."
```

Comment:
With all due respect, I don't agree with the requested changes for the following reasoning:

If we want to empathize with users, we should consider the mindset of a beginner who is new to the website. From their perspective, beginners have no idea whether or not the content displayed on the web page is sorted by relevance(although you might assume that they should know the website only promotes relevant posts by default.). And that is because it is not mentioned anywhere on the web page (if I did miss that, please let me know.). As a newbie, all they see is a list of posts and a message that tells them what they can do with the posts if they were signed in: sorting by relevance, top, latest. The current description only implies that users can sort posts either by top or latest after signing in, disregarding the third option.

Moreover, as a new user of the website, i think it is better to let me know the available options for sorting the posts after I sign in. This approach will create some sort of curiosity for the newcomers, which will increase their likelihood of signing in to see what they can do with the content being displayed on the webpage.

That's all I can think of at the moment. However, I don't think it generates a significant impact on the website, but it enhances it a bit. After all, it is up to you to decide what is better for the project as a maintainer. But, I just wanted to shed some light on the perspectives of beginners.

key takeaway:

* Either display all the available options for users or omit them to increase the chance for new users to sign up for the website. Hence, I am suggesting the following:

```sh
Sign in for the ability to sort posts.
```

or

```sh
Sign in for the ability to sort posts by relevant, latest, or top.
```

What do you think?

Reply:
Thanks for clarification ❤️ 
Looking back to the inception #8932, I need to see conversion rate on this CTA, but I do not have any access to site-wide analytics on dev.to or other public Forem instances. Initial implementation in this PR looks good to me now 🙏